src/main/java/com/hillel/items_exchange/service/UserService.java

```diff
@@ -57,4 +67,29 @@ public boolean existsByEmail(String email) {
     private UserDto mapUserToDto(User user) {
         return modelMapper.map(user, UserDto.class);
     }
+
+    public List<ChildDto> getChildren(User parent) {
+        return convertToDto(parent.getChildren(), ChildDto.class);
+    }
+
+    public void addChildren(User parent, List<ChildDto> childrenDtoToAddList) {
+        final List<Child> childrenToSaveList = new ArrayList<>(convertToModel(childrenDtoToAddList, Child.class, ArrayList::new));
+        childrenToSaveList.forEach(child -> child.setUser(parent));
+        childRepository.saveAll(childrenToSaveList);
+    }
+
+    public void updateChildren(User parent, List<ChildDto> childrenDtoToUpdateList) {
+        final List<Child> childrenToUpdate = new ArrayList<>(convertToModel(childrenDtoToUpdateList, Child.class, ArrayList::new));
+        childrenToUpdate.forEach(child -> child.setUser(parent));
```

Comment:
Two lines above are two times appear in such class - extract it to the new method.